src/App.js

```diff
@@ -29,14 +70,26 @@ class App extends Component {
       latestChannelVersions: null,
       statuses: initStatuses()
     }
-    fetch('https://pollbot.dev.mozaws.net/v1/firefox/ongoing-versions')
-      .then(resp => resp.json())
+
+    this.refreshIntervalId = null;
+  }
+
+  componentDidMount() {
+    fetchOngoingVersions()
       .then(data => {
         this.setState({ latestChannelVersions: data })
       })
       .catch(err =>
         console.error('Failed getting the latest channel versions', err)
-      )
+      );
+    // Setup auto-refresh.
+    this.refreshIntervalId = setInterval(this.refreshStatus.bind(this), 5000);
```

Comment:
I don't think you need to bind to `this` here, as `refreshStatus` is an arrow function (that automatically binds to `this` iiuc).